playbooks/common/openshift-etcd/scaleup.yml

```diff
@@ -53,3 +56,25 @@
     retries: 3
     delay: 30
     until: scaleup_health.rc == 0
+
+- name: Update master etcd client urls
+  hosts: oo_masters_to_config
+  handlers:
+  - include: ../../../roles/openshift_master/handlers/main.yml
+    static: yes
+  roles:
+  - role: openshift_master_facts
+    openshift_master_etcd_hosts: "{{ hostvars
+                                     | oo_select_keys(groups['oo_etcd_to_config'] | union(groups['oo_new_etcd_to_config']))
+                                     | oo_collect('openshift.common.hostname')
+                                     | default(none, true) }}"
+    openshift_master_etcd_port: "{{ (etcd_client_port | default('2379')) if (groups.oo_etcd_to_config is defined and groups.oo_etcd_to_config) else none }}"
+  - role: lib_utils
+  post_tasks:
```

Comment:
This should be definitely be part of the `openshift-master` role so one does not have include the handlers. Could you create a new task file in the role and use `include_role` to include the new task file?

Reply:
Using `include_role` is going to be pretty gnarly with the cert role dependencies on `openshift_master` but I'm going for it anyway since they should be gone soon.

Reply:
Did!